homeassistant/components/timer/services.yaml

```diff
@@ -34,3 +34,18 @@ finish:
   target:
     entity:
       domain: timer
+
+change:
+  name: Change
+  description: Change a timer
+  target:
+    entity:
+      domain: timer
+  fields:
+    duration:
+      description: Duration to add or subtract to the running timer
+      default: 0
+      required: true
+      example: "00:01:00 or 60"
```

Comment:
I recommend showing a negative change as well (i.e. -20) to demonstrate the reduction of time.